docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:10.5
+    container_name: djangoplicity-actions-db
+    restart: always
+    environment:
+      POSTGRES_DB: djangoplicity-actions
+      POSTGRES_USER: djangoplicity-actions
+      POSTGRES_PASSWORD: djangoplicity-actions
+      POSTGRES_HOST_AUTH_METHOD: trust
+    volumes:
+      - ./pgdata:/var/lib/postgresql/data
+    ports:
+      - "5433:5432"
+
+  web:
+    build: .
+    container_name: djangoplicity-actions-web
+    command: python manage.py migrate
+    command: python manage.py runserver 0.0.0.0:8000
```

Comment:
You can use `command: ['./scripts/command-dev.sh']`, that's why that file is included

Reply:
Right, also because multiple commands here doesn't work.